Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -217,4 +217,15 @@ - (void)configureWithData:(id)data {
     NSAssert(false, @"BPKCalendarCell configureWithData: should be overridden. The base class method should never be called.");
 }
 
+- (NSString *)formattedDate:(NSDate *)date locale:(NSLocale *)locale {
+    NSDateFormatter *dateFormatter = [NSDateFormatter new];
```

Comment:
Formatters are, supposedly, expensive to create. I haven't looked at this in a long time, but worth doing some research before we decided to create them in a hot path like this.

Reply:
This is actually no different to how the code worked before, but I'm sure there's some optimisation that can be done 👍

Reply:
I've optimised this by making the calendar provide it's own formatter